tests/manage/z_cluster/cluster_expansion/test_add_capacity_entry_exit_criteria.py

```diff
@@ -301,54 +315,66 @@ def test_add_capacity(
 
         logger.info("Exit criteria verification Success: osd tree verification success")
 
-        # Make sure new pvcs and pods can be created and IOs can be run from
-        # the pods
-        num_of_pvcs = 1
-        rwo_rbd_pods = multi_dc_pod(
-            num_of_pvcs=num_of_pvcs,
-            pvc_size=5,
-            project=project,
-            access_mode="RWO",
-            pool_type="rbd",
-        )
-        rwo_cephfs_pods = multi_dc_pod(
-            num_of_pvcs=num_of_pvcs,
-            pvc_size=5,
-            project=project,
-            access_mode="RWO",
-            pool_type="cephfs",
-        )
-        rwx_cephfs_pods = multi_dc_pod(
-            num_of_pvcs=num_of_pvcs,
-            pvc_size=5,
-            project=project,
-            access_mode="RWX",
-            pool_type="cephfs",
-        )
-        # Create rwx-rbd pods
-        pods_ios_rwx_rbd = multi_dc_pod(
-            num_of_pvcs=num_of_pvcs,
-            pvc_size=5,
-            project=project,
-            access_mode="RWX-BLK",
-            pool_type="rbd",
-        )
-        cluster_io_pods = (
-            rwo_rbd_pods + rwo_cephfs_pods + rwx_cephfs_pods + pods_ios_rwx_rbd
-        )
-
-        with ThreadPoolExecutor() as pod_ios_executor:
-            for p in cluster_io_pods:
-                if p.pod_type == "rbd_block_rwx":
-                    logger.info(f"Calling block fio on pod {p.name}")
-                    pod_ios_executor.submit(cluster_exp_helpers.raw_block_io, p, "100M")
-                else:
-                    logger.info(f"calling file fio on pod {p.name}")
-                    pod_ios_executor.submit(p.run_io, "fs", "100M")
-
-        for pod_io in cluster_io_pods:
-            pod_helpers.get_fio_rw_iops(pod_io)
-
+        # Make sure new pvcs and pods can be created and IOs can be run from the pods
+        logger.info("Start creating new PVCs and pods, and run IO from the pods")
+        if config.ENV_DATA["platform"].lower() in [
+            constants.VSPHERE_PLATFORM,
+            constants.IBMCLOUD_PLATFORM,
+        ]:
+            # Change the method of creating resources when we use vSphere and IBM Cloud platforms
+            self.sanity_helpers.create_resources(
```

Comment:
checking to understand; is it intentional that we don't run IO on these resources?